src/Core/Analysis/IExclusionSettingsFileStorage.cs

```diff
@@ -0,0 +1,30 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarQube.Client.Models;
+
+namespace SonarLint.VisualStudio.Core.Analysis
+{
+    public interface IExclusionSettingsFileStorage
```

Comment:
Based on Ugras' PR

Reply:
@ugras-ergun-sonarsource do you mean you'll push the change to 'branch-exclusions`, or you'll create a PR?

Reply:
I have a pr already. I had made some changes to signatures locally but I've reverted it back and in the confusion, I have created noise you can disregard my comment.